hazelcast/src/test/java/com/hazelcast/client/map/ClientMapReplaceAllTest.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.client.map;
+
+import com.hazelcast.client.config.ClientConfig;
+import com.hazelcast.client.test.TestHazelcastFactory;
+import com.hazelcast.config.Config;
+import com.hazelcast.config.MapConfig;
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.internal.util.UuidUtil;
+import com.hazelcast.map.IMap;
+import com.hazelcast.test.HazelcastSerialClassRunner;
+import com.hazelcast.test.HazelcastTestSupport;
+import com.hazelcast.test.annotation.SlowTest;
+import com.hazelcast.test.starter.HazelcastStarter;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import testsubjects.StaticSerializableBiFunction;
+import testsubjects.StaticSerializableBiFunctionEx;
+
+import static org.junit.Assert.assertEquals;
+
+@RunWith(HazelcastSerialClassRunner.class)
+@Category(SlowTest.class)
+public class ClientMapReplaceAllTest extends HazelcastTestSupport {
+
+    private HazelcastInstance memberInstance;
+
+    protected TestHazelcastFactory hazelcastFactory = new TestHazelcastFactory();
+
+    protected HazelcastInstance clientInstance;
+
+    @After
+    public final void stopHazelcastInstances() {
+        hazelcastFactory.terminateAll();
+        if (memberInstance != null) {
+            memberInstance.shutdown();
+        }
+    }
+
+    @Before
+    public final void startHazelcastInstances() {
+        Config config = getConfig();
+        MapConfig mapConfig = new MapConfig("mapWithTTL");
+        mapConfig.setTimeToLiveSeconds(1);
+        config.addMapConfig(mapConfig);
+        ClientConfig clientConfig = getClientConfig();
+        memberInstance = HazelcastStarter.newHazelcastInstance("4.0.3");
+        clientInstance = hazelcastFactory.newHazelcastClient(clientConfig);
```

Comment:
Theses tests are failing, You should not use `TestHazelcastFactory`. That is used for skipping the network. In multi version tests where we use `HazelcastStarter`, we don't have that support. You should start the client the usual way. 
`HazelcastClient.newHazelcastClient();`